src/snowflake/snowpark/relational_grouped_dataframe.py

```diff
@@ -348,6 +355,7 @@ def end_partition(self, pdf: pandas.DataFrame) -> pandas.DataFrame:
         _apply_in_pandas_udtf = self._df._session.udtf.register(
             _ApplyInPandas,
             output_schema=output_schema,
+            input_names=input_names,
```

Comment:
Can't we get the input column names from `self._df.schema` similar to how we get the `input_types` here:  https://github.com/snowflakedb/snowpark-python/blob/main/src/snowflake/snowpark/relational_grouped_dataframe.py#L344-L346 ?

Reply:
I agree with Sara. we can infer `input_names` from `self._df.schema` similar to how to do it for `input_types`.

Reply:
Oh somehow I misread the part about `self._df.schema`. In that case it does make sense to do that instead.